src/versions.tf

```
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.0, < 6.0.0"
+      version = ">= 5.0, < 6.3.0"
```

```suggestion
      version = ">= 5.0"
```

I this component works with the v6 AWS provider, we can skip the upper constraint. 
And probably we need to increase low bound to `6.3.0` if that's the minimum version that supports the feature